src/cookies.test.ts

```diff
@@ -0,0 +1,67 @@
+		expect(getCookie('s_vi')).toEqual(
+			'[CS]v1|29AB343C05013259-4000014980021C3D[CE]',
+		);
+	});
+
+	it('should be able to get country cookie', () => {
+		document.cookie = 'GU_geo_country=GB;';
+		expect(getCookie('GU_geo_country')).toEqual('GB');
+	});
+
+	it('should be able to set a session cookie', () => {
+		setSessionCookie('cookie-1-name', 'cookie-1-value');
+		expect(document.cookie).toEqual(
+			'cookie-1-name=cookie-1-value; path=/; domain=.theguardian.com',
+		);
+	});
+
+	it('should be able the clean a list of cookies', () => {
```

Comment:
```suggestion
	it('should be able to clear a list of cookies', () => {
```

Reply:
this now tests the `remove` method instead